src/components/Nav.jsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from 'react-router-dom';
+import profileIcon from '../assets/images/icons/profile.svg';
+import styles from '../css/components/navigation.module.css';
+
+const Nav = () => (
+  <header className={styles.navbar}>
+    <div className={`page-container ${styles.pageContainer}`}>
+      <Link to="/books" className={styles.logo}>
+        Bookstore CMS
+      </Link>
+      <ul className={styles.linksContainer}>
+        <li><Link to="/books" className={`${styles.link} ${styles.activeLink}`}>BOOKS</Link></li>
+        <li><Link to="/categories" className={styles.link}>CATEGORIES</Link></li>
+      </ul>
```

Comment:
- [ ] There should be 2 routes books and Categories and when we click books it should `renders a form with inputs for the title and the author of the book`, `Has a button for removing books, just a place holder`, `Has a button to submit`  the button wouldn't add the book *yet*, this is just a placeholder for now.
- [ ] When we click the `Categories` Should display a button "Check status" only. But is showing empty when I render categories. Kindly fix it.
- [ ] You have good styling the only thing is you need to make the book's routes a default route when I click books links, it is empty at this point. eg:
 `<Route exact path="/">
      <BookList />
    </Route>`

Reply:
Hi @roshan-bajgain, thank you for the feedback but I have a question, where should the button to remove books go? in the form? or on each book component?

Reply:
@Trictonicmp Inside books routes there should be an input field for title and author and make some hardcoded for author and title and each has a removing buttons. Kindly implement it